com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -261,8 +261,14 @@ private struct PacketLossCache
         private NetworkPipeline m_UnreliableSequencedFragmentedPipeline;
         private NetworkPipeline m_ReliableSequencedPipeline;
 
+        /// <summary>
+        /// The client id used to represent the server.
```

Comment:
Might want to add a note that this uses the "transport ID" of the client. It doesn't match the client IDs that users are normally handling.

Reply:
On second thought maybe it's not worth mentioning it. _Every_ client ID handled in this file is a "transport ID". Making a note only here could lead someone to think other uses of client IDs are not "transport IDs".